crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -294,6 +305,12 @@ impl AuthorityPerEpochStore {
                 }
             })
             .collect();
+        if let Some(epoch_start_configuration) = epoch_start_configuration {
```

Comment:
How do we make sure that the config is always there?

Reply:
This config is injected differently for first and other epochs. For the first epoch value from genesis is used, and for all other epochs value is set in reopen_for_next_epoch.

Reply:
I don't see anywhere where we pass None?

Reply:
We do pass `None` on the node start (and then expect it to be delivered via mechanisms described above). Btw, I know this is not ideal and if anyone have suggestions how to make it better I am happy to implement them

Reply:
OK this is reasonable. My only request is then to add a else branch here and assert epoch=0 there, and comment where this will be initialized.

Reply:
I added detailed comment to the line when and how we initialize the value in epoch store DB (but no assert, you can see in comments that we can't just put assert in `else`)